spark2/src/test/java/org/apache/iceberg/examples/SchemaEvolutionTest.java

```diff
@@ -22,18 +22,27 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
```

Comment:
Style: no blocks of imports, just one list ordered alphabetically.

Reply:
done.